lib/api/backbeat/listLifecycleCurrents.js

```diff
@@ -70,7 +70,7 @@ function listLifecycleCurrents(authInfo, request, log, callback) {
                 IsTruncated: false,
             };
             return handleResult(listParams, requestMaxKeys, authInfo,
-                bucketName, emptyList, log, callback);
+                bucketName, emptyList, false, log, callback);
```

Comment:
The bucket itself could be versioned here, although we pass `false`, not sure this is wanted?

Reply:
Although the listing is empty, I will still move the versioning check earlier and introduce a variable for the possibility of using it in future implementations.